cmd/bots/telegram/telegram.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"os/signal"
+
+	"github.com/pkg/errors"
+	"github.com/procyon-projects/chrono"
+	"nodemon/cmd/bots/internal/common"
+	initial "nodemon/cmd/bots/internal/common/init"
+	"nodemon/cmd/bots/internal/common/messaging/pair"
+	"nodemon/cmd/bots/internal/common/messaging/pubsub"
+	"nodemon/cmd/bots/internal/telegram/config"
+	"nodemon/cmd/bots/internal/telegram/handlers"
+	pairResponses "nodemon/pkg/messaging/pair"
+)
+
+func main() {
+	err := runTelegramBot()
+	if err != nil {
+		switch err {
+		case context.Canceled:
+			os.Exit(130)
```

Comment:
Is it ok to return non zero exit code? If user interrupted process manually I think it should be considered as graceful shutdown with zero exit code.

Reply:
Check the messages above